Charts/Classes/Renderers/ChartYAxisRenderer.swift

```diff
@@ -70,6 +70,10 @@ public class ChartYAxisRenderer: ChartAxisRendererBase
         
         let rawInterval = range / Double(labelCount)
         var interval = ChartUtils.roundToNextSignificant(number: Double(rawInterval))
+        if(_yAxis.valueFormatter?.maximumFractionDigits == 0)
```

Comment:
This is not about limit lines, right? It's your solution for the formatter issue. Should be another PR if think it's good to everyone